test/float8/test_fsdp2/test_fsdp2.py

```diff
@@ -18,7 +18,7 @@
 from torchao.float8.config import CastConfig, Float8LinearConfig, ScalingType
 from torchao.float8.float8_linear_utils import convert_to_float8_training
 from torchao.float8.fsdp_utils import WeightWithDynamicFloat8CastTensor
-from fsdp2_common import check_parity_bf16_mp, check_parity_no_mp
+from torchao.testing.float8_fsdp2_utils.float8 import check_parity_bf16_mp, check_parity_no_mp
```

Comment:
just curious, why is this `torchao.testing.float8_fsdp2_utils.float8` instead of `torchao.testing.float8_fsdp2_utils`?

Reply:
sorry.. it was a typo. I just updated the pr again. It will be "torchao.testing.float8.fsdp2_utils"